doc/alarm.md

```diff
@@ -184,17 +188,30 @@ public interface SmsSender {
 
 **C. Webhook alarm service**
 
-Webhook alarm service is the api of Pinpoint alarm over HTTP. To receive alarms you will need to prepare the receiver server. There is an example of webhook slack receiver project, see 3. Others - [5) An example of webhook receiver project](##3.Others). The payloads of the webhook have the different schemas depending on the alarm checker type. you can see the example of the payload schemas in 3.Others - [6) The Specification of webhook payloads and the examples](##3.Others). If you have an previous release of pinpoint-2.1.1, you should change the schema of alarm_rule table.
+Webhook alarm service is the API of Pinpoint alarm over HTTP. 
+
+**To receive alarms, you will need to set up the receiver server, or You can use an example of webhook slack receiver project. see [An example of webhook receiver project](https://github.com/doll6777/slack-receiver).**
+The alarm message(payloads) of the webhook have the different schema depending on the alarm checker type. 
+You can see the payload schemas in [3.Others - The Specification of webhook payloads and the examples](##3.Others). 
+
+To enable the webhook alarm service, 
+You need to set *webhook.enable* and *webhook.receiver.url* in [batch-root.properties](https://github.com/pinpoint-apm/pinpoint/blob/master/batch/src/main/resources/batch-root.properties) file.
```

Comment:
usually use `configure` instead of `set`

Reply:
I should fix the line 121 as well.